raiden/tests/fixtures/variables.py

```diff
@@ -181,7 +182,27 @@ def blockchain_private_keys(blockchain_number_of_nodes, blockchain_key_seed):
 @pytest.fixture(scope='session')
 def port_generator(request):
     """ count generator used to get a unique port number. """
-    return count(request.config.option.initial_port)
+
```

Comment:
Yep, it has nothing to do with the PR, I just needed that for my personal setup and was too lazy to open a new one